DiffractometerMockup.py

```diff
@@ -73,7 +73,82 @@ def init(self):
         self.image_width = 400
         self.image_height = 400
 
-        self.equipment_ready()
+        self.equipmentReady()
+        self.user_clicked_event = AsyncResult()
+
+        self.phiMotor = self.getDeviceByRole('phi')
```

Comment:
Sorry, I am not happy about this part. It is fine to add some hardware objects, but could it be implemented via GenericDiffractometer?